content/index.rst

```diff
@@ -10,5 +10,4 @@ Odoo Documentation
     applications
     administration
     developer
-    services
```

Comment:
ALT already had something to say about this but I want to add that a page about consultancy services also needs to be brought back into this sub-doc. It had been lost during the doc apocalypse. See task https://www.odoo.com/web?debug=0#id=2585131&action=333&active_id=3835&model=project.task&view_type=form&cids=1&menu_id=5195

Reply:
No, this page is outdated, and it should not be in the doc.
Instead, we should refer to the Odoo Implementation Book: https://www.odoo.com/r/Implementation_methodology